docs/src/man/reshaping_and_pivoting.md

```diff
@@ -380,3 +380,49 @@ julia> first(unstack(x, :Species, :vsum), 6)
 │ 4   │ PetalWidth  │ 0.244       │ 1.326           │ 2.026          │
 │ 5   │ id          │ 25.5        │ 75.5            │ 125.5          │
 ```
+
+To turn an `AbstractDataFrame` on its side, use [`permutedims`](@ref).
+
+```jldoctest reshape
+julia> df1 = DataFrame(a=["x", "y"], b=[1.,2.], c=[3,4], d=[true,false])
+2×4 DataFrame
+│ Row │ a      │ b       │ c     │ d    │
+│     │ String │ Float64 │ Int64 │ Bool │
+├─────┼────────┼─────────┼───────┼──────┤
+│ 1   │ x      │ 1.0     │ 3     │ 1    │
+│ 2   │ y      │ 2.0     │ 4     │ 0    │
+
+julia> permutedims(df1)
+3×3 DataFrame
+│ Row │ a      │ x       │ y       │
+│     │ String │ Float64 │ Float64 │
+├─────┼────────┼─────────┼─────────┤
+│ 1   │ b      │ 1.0     │ 2.0     │
+│ 2   │ c      │ 3.0     │ 4.0     │
+│ 3   │ d      │ 1.0     │ 0.0     │
+```
+
+Note that the first column (by default) of the original `df`
+becomes the column names in the permuted result,
+and the column names of the original become a new column.
+Note also that the types of the other columns
+are the result of `promote_type` on _all_ the permuted columns.
+
+```jldoctest reshape
```

Comment:
I was thinking that, since it's not recommended, but may be the result of many of this type of operation, it would be worth pointing out explicitly, no?

A thought I had was to include example code for fixing the problem, but the manual didn't seem like the right place for it.

Reply:
I am OK with this example. I would just mention in the text above that a typical use case is for columns with homogenous eltype and we show here a case what happens when it is not the case.

Reply:
Is this still live - @nalimilan do you still object, or are you OK with it?

Reply:
I am OK with what we have now. Manual can be easily improved later if needed.